### PR TITLE
Date and time logging plus a shutdown confirmation system

### PR DESCRIPTION
**Pull request by OzzyHelix**
After creating your pull request, tick these boxes if they are applicable to you.

- [x] I have tested my changes against the `review` branch (the latest developmental version), and this pull request is targeting that branch as a base
- [x] I have tested my changes on Python 3.5/3.6

----

### Description
this will write logs with the time and date of when they are used as shown here 
image
the logs will also delete after a days amount of secs   
in this pull request is an extension to the shutdown command
to prevent the bot from being shutdown by human error  
it confirms with the user that they want the bot to be shutdown
image example:
image

### Related issues (if applicable)
of course the bot needs to be restarted to active log deletion

### CODE REVIEW

**Comment by OzzyHelix** [comment]
forgive me I decided to redo my original pull request

**Comment by OzzyHelix** [comment]
I made a mistake a while back  I had tried to go back to an earlier commit and it messed up the ability to merge

**Comment by OzzyHelix** [comment]
ok I fixed most of the problems that arose from changes between updates

File: musicbot/bot.py [hunk 1]
```diff
@@ -2449,15 +2461,35 @@ def check(message):
...
-        await self.safe_send_message(channel, "\N{WAVING HAND SIGN}")
-        
-        player = self.get_player_in(channel.guild)
-        if player and player.is_paused:
-            player.resume()
-        
-        await self.disconnect_all_voice_clients()
-        raise exceptions.TerminateSignal()
+    # This uses a lot of the search command selection functionality
+    async def cmd_shutdown(self, channel, message):
```
Comment (MattBSG) [inline musicbot/bot.py:2465]: Can you please update to use discord.py rewrite syntax? https://discordpy.readthedocs.io/en/rewrite/migrating.html

**Review by MattBSG — CHANGES_REQUESTED** [review]

**Comment by OzzyHelix** [comment]
I wish to clarify the rewrite syntax is to use guild instead of server or is there more? the rewrite has me a bit confused I will continue working while I wait for an answer

File: musicbot/bot.py [hunk 2]
```diff
@@ -2449,15 +2461,35 @@ def check(message):
...
+            await warning_message.add_reaction(r)
+
+        try:
+            reaction, user = await self.wait_for('reaction_add', timeout=30.0, check=check)
+        except asyncio.TimeoutError:
+            await self.safe_delete_message(warning_message)
+            return
+
+        for r in reactions:
+            await warning_message.add_reaction(r)
```
Comment (Chaos21) [inline musicbot/bot.py:2484]: There's an attribute error at this line.

'Message' has no attribute 'add_reaction'

**Pull request closed without merging**